src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedRelation.java

```diff
@@ -62,9 +44,10 @@ public FlaggedRelation(final Relation relation)
     public JsonObject asGeoJsonFeature(final String flagIdentifier)
     {
         final JsonObject feature = this.relation.asGeoJsonGeometry();
-        final JsonObject featureProperties = feature.getAsJsonObject("properties");
```

Comment:
This would return null and fail some tests after https://github.com/osmlab/atlas/pull/369